superset-frontend/src/explore/components/DataTablesPane.tsx

```diff
@@ -191,11 +195,13 @@ export const DataTablesPane = ({
       if (data[type]?.length === 0) {
         return <span>No data</span>;
       }
+      const renderWithPagination = true;
       return (
         <TableView
           columns={columns[type]}
           data={filteredData[type]}
-          withPagination={false}
+          withPagination={renderWithPagination}
```

Comment:
```suggestion
          withPagination
```
^ I think this is what @villebro is suggesting... does this still throw errors?

Reply:
yep, this one works fine. thanks!